### PR TITLE
[Snort] Add Snort Integration

### LINKED ISSUE #937: Migrate Snort module
# Integration release checklist

This checklist is intended for integrations maintainers to ensure consistency
when creating or updating a Package, Module or Dataset for an Integration.

### Migration Checklist

- [ ] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [ ] Supported versions of the monitoring target are documented
- [ ] Generated output for at least 1 log file exists
- [ ] Documentation explains how to configure monitoring target to work with the integration
- [ ] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [ ] At least a manual test with ES / Kibana / Agent has been performed.
- [ ] Required Kibana version set to:
- [ ] Screenshot of the "Add Integration" page on Fleet added to PR

### Dashboards

- [ ] Dashboards exists
- [ ] Screenshots added to manifest
- [ ] Data stream filters added to visualizations

### Data Streams
- snort
  - [ ] [Pipeline tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/pipeline_testing.md) added
  - Inputs
    - file
    - tcp
    - udp
      - [ ] [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) added

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Add Snort integration

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Author's Checklist

- [ ]

## How to test this PR locally

```
cd integrations/packages/snort
elastic-package stack down && elastic-package build && elastic-package stack up -d -v -p default && eval "$(elastic-package stack shellinit)" && elastic-package test  -v
```

## Related issues

- Closes #937

### CODE REVIEW

File: packages/snort/_dev/build/build.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@1.11
```
Comment (legoguy1000) [inline packages/snort/_dev/build/build.yml:3]: Need to update to 1.12

File: packages/snort/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 2]
```diff
@@ -0,0 +1,200 @@
+---
+description: Pipeline for parsing Snort logs
+processors:
+  - set:
+      field: event.ingested
+      value: '{{_ingest.timestamp}}'
+  - set:
+      field: ecs.version
+      value: '1.11.0'
```
Comment (legoguy1000) [inline packages/snort/data_stream/log/elasticsearch/ingest_pipeline/default.yml:9]: Need to update to 1.12

**Review by P1llus — APPROVED** [review]
If you could bump minimum version to 7.16 I could merge right away

**Pull request merged**